Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM sameersbn/redmine:3.4.10
+FROM redmine:3.4
```

Comment:
Please use the full version tag

Reply:
I'd prefer to stay on latest patch level and using automated builds. Upstream containers are usually well tested.

Reply:
IMHO we can use 3.4 as we are using even latest in some containers and staying on the latest version of a fixed major (f.e. 3.4) version looks like a good compromise to me.